frontend/www/js/omegaup/arena/arena.js

```diff
@@ -77,6 +79,13 @@ export function GetOptionsFromLocation(arenaLocation) {
       options.preferredLanguage = payload.preferred_language || null;
     }
   }
+
+  options.headerPayload = null;
+  if (document.getElementById('header-payload') !== null) {
```

Comment:
```suggestion
  const headerPayloadElement = document.getElementById('header-payload');
  if (headerPayloadElement !== null) {
```

por consistencia con L73. y una pregunta relacionada, será posible meter todo lo que actualmente está en `header-payload` en `payload` para sólo tener un payload?

Reply:
Listo el cambio.

Con respecto a meter todo en el `payload` estoy pensando que podemos hacer un refactor, sólo que prefiero hacerlo en un cambio extra porque este crecería demasiado.

Reply:
según yo debería ser posible hacerlo en este cambio sin que crezca mucho. después se puede hacer el refactor para que quede más bonito.

Reply:
A ver que te parece la solución que implementé:

En la función `UIToolls::render` obtengo `smartyProperties['payload']` y combino el array con el array de headerPayload para asignarlo a la variable `payload` de smarty. 

Aún no puedo quitar la variable de smarty `headerPayload` porque creo que dependemos de que todos los archivos ya manden llamar la función `render`, más aparte otras cosas en los archivos `.tpl`.